Formula/rust.rb

```diff
@@ -2,16 +2,16 @@ class Rust < Formula
   desc "Safe, concurrent, practical language"
   homepage "https://www.rust-lang.org/"
   license any_of: ["Apache-2.0", "MIT"]
-  revision 1
+  revision 2
```

Comment:
I’m uncertain whether this revision bump is justified. It seemed like the more-conservative option was to bump it, but I’ll happily defer to Homebrew maintainers’ judgment.